etc/afpd/auth.h

```diff
@@ -31,11 +31,7 @@ static const struct afp_versions  afp_versions[] = {
 #define USERIBIT_ALL   (USERIBIT_USER | USERIBIT_GROUP | USERIBIT_UUID)
 
 extern uid_t    uuid;
-#if defined( sun ) && !defined( __svr4__ ) || defined( ultrix )
```

Comment:
I think the `__svr4__` macro is used for Solaris, which we do still support.

Reply:
Oops, I didn't notice the negative check on that macro! Please by all means get rid of this.